### PR TITLE
Add Travis-CI yml

### LINKED ISSUE #19: Travis build?
Would you be interested in hooking this project up to Travis? (I tried to add a simple .travis.yml file, but some dependencies didn't get resolved ...)

https://travis-ci.org/gliptak/scalatest-maven-plugin/jobs/94070065

### PR DESCRIPTION
**Pull request by cstroe**
This PR adds a `.travis.yml` file to run the maven integration tests in a Travis CI build.  You can then easily enable the Travis-CI integration for this repo and have Travis-CI run builds for every PR and commit on master.

An example of a passing build is [here](https://travis-ci.org/cstroe/scalatest-maven-plugin/builds/235508351).

Fixes #19

### CODE REVIEW

**Comment by cstroe** [comment]
@gcberger Any interest in getting this merged?

**Comment by rdefreitas** [comment]
@bvenners here

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+language: java
+
+sudo: false
+
+jdk:
+  - oraclejdk8
```
Comment (katrinsharp) [inline .travis.yml:6]: Should we perhaps use higher java version? There is outstanding PR for supporting java 11...

Comment (cstroe) [reply .travis.yml:6]: Sure, we can do that.  In this case, we'd have to wait for Java 11 support to be added before using it.

We can also run tests with each JDK.

In either case, I think we should still run tests against JDK 8, so we should merge this PR and have another one to add a JDK 11 build.

**Comment by katrinsharp** [comment]
Sounds like a plan to me.

**Comment by katrinsharp** [comment]
@cstroe were you planning to add a build as another PR? If not, I probably should do it. Please let me know. Thanks.

**Pull request merged**